src/JWT.ts

```diff
@@ -165,6 +165,11 @@ export const SUPPORTED_PUBLIC_KEY_TYPES: PublicKeyTypes = {
      *   not an ethereumAddress
      */
     'EcdsaPublicKeySecp256k1',
+    /**
+     * @deprecated, supported for backward compatibility. Equivalent to EcdsaSecp256k1VerificationKey2019 when key is
```

Comment:
comments here are not correct. please reference it to the verifiable conditions verification method type

Reply:
removed comments